### ISSUE TITLE
Yolov3 training loss 

### ISSUE THREAD

**Issue by sainivedh19pt** [open-mmlab/mmdetection#7087]
On my custom dataset, the training loss isn't converging (hovering around 20-30)...the mAP is also behaving weirdly..after 1 epoch its 0.5, then 0.3, 0.2, 0.1... 

I tried same dataset with faster_rcnn config it's working as expected..

Any runtime/scheduler/train_pipeline that I must use differently for Yolov3 ??

**Comment by sainivedh19pt** [comment]
Training has stabilised after some epochs..achieved recall of 0.98, ap@0.5 of 0.8 at 100 epochs

**Comment by David-Biggs** [comment]
> , then 0.3, 0.2, 0

Did you manage to figure out what was wrong? I am having a similar issue. My training losses are quite large (> 500) and my mAP are all 0.

**Comment by AbdelazizHamadi** [comment]
Hey @David-Biggs I'm having the same issue i coudn't see why it's happening, have u found the solution for this problem ?
